keyboards/lily58/keymaps/perugi/keymap.c

```diff
@@ -77,39 +77,39 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
  * |------+------+------+------+------+------|   {   |    |    }  |------+------+------+------+------+------|
  * |   _  |   !  |   @  |   #  |   $  |   %  |-------|    |-------|   ^  |   &  |   *  |   (  |   )  |   +  |
  * `-----------------------------------------/       /     \      \-----------------------------------------'
- *                   | LAlt | LGUI |LOWER | /Space  /       \Enter \  |RAISE |BackSP| SLO  |
+ *                   | LAlt | LGUI |LOWER | /Space  /       \Backsp\  |RAISE |BackSP| SLO  |
```

Comment:
~ is missing from your keymap. While this does not concern this particular branch, it would be wasteful to open a new issue just for this.

Reply:
Added a ~ symbol to the keymap on the raised layer. See commit 25618fd